### PR TITLE
added entries for definition and submission

### PR DESCRIPTION
**Pull request by awoie**
@csuwildcat @glcohen @timcappalli  are there any other potentially top level claims that we need to register? Currently, I identified `presentation_definition` and `presentation_submission` but I guess `input_descriptor` and `presentation_requirements` might be top-level too?

### CODE REVIEW

**Review by csuwildcat — APPROVED** [review]

File: spec/spec.md [hunk 1]
```diff
@@ -2033,3 +2033,25 @@ JSONPath                      | Description
...
+This specification registers the claims in section [Registry Contents]() in the IANA JSON Web Token Claims registry defined in [[RFC7519]].
+
+##### Registry Contents
+
+Presentation Definition | Values
+------------------------------|------------
+Claim Name: | `presentation_definition`
+Claim Description: | Presentation Definition
+Change Controller: | DIF Claims & Credentials - Working Group - TBD@email.com
+Reference | Section XXX of this document
```
Comment (brentzundel) [inline spec/spec.md:2050]:
```suggestion
Reference | Section 5 of this document
```

File: spec/spec.md [hunk 2]
```diff
@@ -2033,3 +2033,25 @@ JSONPath                      | Description
...
+
+Presentation Submission                      | Values
+------------------------------|------------
+Claim Name: | `presentation_submission`
+Claim Description: | Presentation Submission
+Change Controller: | DIF Claims & Credentials - Working Group - TBD@email.com
+Specification Document(s): | Section XXX of this document
```
Comment (brentzundel) [inline spec/spec.md:2057]:
```suggestion
Specification Document(s): | Section 6 of this document
```

File: spec/spec.md [hunk 3]
```diff
@@ -2033,3 +2033,25 @@ JSONPath                      | Description
...
+
+This specification registers the claims in section [Registry Contents]() in the IANA JSON Web Token Claims registry defined in [[RFC7519]].
+
+##### Registry Contents
+
+Presentation Definition | Values
+------------------------------|------------
+Claim Name: | `presentation_definition`
+Claim Description: | Presentation Definition
+Change Controller: | DIF Claims & Credentials - Working Group - TBD@email.com
```
Comment (brentzundel) [inline spec/spec.md:2049]:
```suggestion
Change Controller: | DIF Claims & Credentials - Working Group - https://github.com/decentralized-identity/claims-credentials/blob/main/CODEOWNERS

```

File: spec/spec.md [hunk 4]
```diff
@@ -2033,3 +2033,25 @@ JSONPath                      | Description
...
+Reference | Section XXX of this document
+
+Presentation Submission                      | Values
+------------------------------|------------
+Claim Name: | `presentation_submission`
+Claim Description: | Presentation Submission
+Change Controller: | DIF Claims & Credentials - Working Group - TBD@email.com
```
Comment (brentzundel) [inline spec/spec.md:2056]:
```suggestion
Change Controller: | DIF Claims & Credentials - Working Group - https://github.com/decentralized-identity/claims-credentials/blob/main/CODEOWNERS

```

**Comment by glcohen** [comment]
Hi @awoie is this in a state to be merged?

**Comment by awoie** [comment]
> Hi @awoie is this in a state to be merged?

Yes, the PR is fine if the only claims that we want to register are `presentation_definition` and `presentation_submission`. Isn't it also required to add `input_descriptors` and `presentation_requirements`?

**Comment by csuwildcat** [comment]
@awoie I don't understand - `input_descriptors` and `presentation_requirements` are sub-objects within `presentation_definition` and `presentation_submission`, so don't they fall under those top-level registrations?

**Comment by awoie** [comment]
> @awoie I don't understand - `input_descriptors` and `presentation_requirements` are sub-objects within `presentation_definition` and `presentation_submission`, so don't they fall under those top-level registrations?

then, we don't need to register them. feel free to merge the PR.

**Review by csuwildcat — APPROVED** [review]

**Pull request merged**